index.html

```diff
@@ -41,6 +45,177 @@ <h1 class="headline-heading">I'm Usman <br> Glad to see you!</h1>
             </ul>
         </section>
     </section>
+    <!-- Headline Section Ends Here -->
+
+    <!-- Works Section Starts Here -->
+    <!-- Card 1 starts -->
+
+    <section class="works-card">
+        <img src="./assets/works-1.png" alt="Recent Work">
+        <h1 class="works-card-heading-large">Tonic</h1>
```

Comment:
It looks nice with the help of CSS, but for better accessibility, it's required to have one and only one `h1` tag on every page, you can use `h[2-6]` tags in here, my recommendation is `h3` 😊.